### PR TITLE
Add autograded tests for unit 1

### PR DESCRIPTION
**Pull request by abolinsky**

### CODE REVIEW

File: 01_objects/01_challenge/traffic_light.h [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#pragma once
```
Comment (dancer-yuma) [inline 01_objects/01_challenge/traffic_light.h:1]: ASK: Do we need to implement a header file? I think a learner is supposed to implement only a cpp file

File: 01_objects/02_challenge/rectangle.h [hunk 2]
```diff
@@ -0,0 +1 @@
+// TODO
```
Comment (dancer-yuma) [inline 01_objects/02_challenge/rectangle.h:1]: ASK: Do we need to implement a header file? I think a learner is supposed to implement only a cpp file.

File: 01_objects/03_challenge/bank_account.h [hunk 3]
```diff
@@ -0,0 +1 @@
+// TODO
```
Comment (dancer-yuma) [inline 01_objects/03_challenge/bank_account.h:1]: ASK: Do we need to implement a header file? I think a learner is supposed to implement only a cpp file.

**Review by dancer-yuma — COMMENTED** [review]
@abolinsky 
Question: why are learners asked to implement a `.h` file in this unit? Is this necessary for tests?

**Comment by abolinsky** [comment]
@dancer-yuma resolved all issues!

**Review by dancer-yuma — APPROVED** [review]
@abolinsky 
Looks good to me!

**Pull request merged**